### ISSUE TITLE
need help

### ISSUE THREAD

**Issue by xiyang2002** [YHYHYHYHYHY/ATFNet#5]
作者你好，贵团队在表9的数据结果是自己实验测试的吗? 回望窗口是一致的吗？为什么Itransformer和DLinear模型数据和原文不一致呢？

**Comment by YHYHYHYHYHY** [comment]
The results displayed in Table 9 were obtained through our own experiments. Considering that different models may have varying optimal input lengths in different settings (dataset and forecasting horizon), for each setting we conduct experiments with three distinct look-back window size $L\in\{96, 192, 336\}$ and always choose the best results to create strong baselines and ensure a fairer comparison. 

We did not directly employ the reported results from the original papers of baselines, and the differences observed between our results and the reported results can be attributed to specific hyperparameter settings (including the look-back window) in each experiment. Since conducting a grid search for each baseline proves to be excessively time-consuming, we adopt the following experimental approach to strike a balance between fairness and efficiency:

Regarding the model hyperparameters (such as model dimension and layers of Encoder), we utilize the default settings provided in their respective code repositories. As for the training hyperparameters (such as batch size and learning rate), we employ a unified configuration across all baselines.

We have provided scripts under `./scripts/` for you to reproduce the results (including all baselines) in our paper. If you are specifically interested in reproducing the results from the original papers of the baselines, we recommend referring to the official code repository of each baseline and following their provided experiment settings using the accompanying scripts.

We hope this response adequately addresses your concern. If you have any further questions or require additional assistance, feel free to contact us.
